tools/ua.bash

```diff
@@ -1,19 +1,14 @@
 # bash completion for ubuntu-advantage-tools
 
-SERVICES=$(python3 -c "
-from uaclient.entitlements import valid_services
-from uaclient.config import UAConfig
-cfg = UAConfig()
-print(*valid_services(cfg=cfg), sep=' ')
-")
-
 _ua_complete()
 {
-    local cur_word prev_word
+    local cur_word prev_word services subcmds base_params
     cur_word="${COMP_WORDS[COMP_CWORD]}"
     prev_word="${COMP_WORDS[COMP_CWORD-1]}"
 
-    subcmds=$(pro --help | awk '/^\s*$|Available|Use/ {next;} /Flags:/{flag=1;next}/Use ubuntu-avantage/{flag=0}flag{ if ( $1 ~ /,/ ) { print $2} else print $1}')
+    services=$(python3 -c "from uaclient.entitlements import valid_services; from uaclient.config import UAConfig; print(*valid_services(cfg=UAConfig()), sep=' ')
+")
+    subcmds=$(pro --help | awk '/^\s*$|^\s{5,}|Available|Use/ {next;} /Flags:/{check=1;next}/Use ubuntu-avantage/{check=0}check{ if ( $1 ~ /,/ ) { print $2} else print $1}')
```

Comment:
one day I hope can we introduce something like `pro autocomplete` that does all this instead of an ad-hoc inline python script and awk-ing the help output

until then I think this change makes sense

Reply:
Is your intention to say that a `awk` against the help output is some kind of ugly undesired workaround?
You're totally right.
`pro autocomplete` can be worked on in the future for sure.

Reply:
yes that is what I was trying to say :upside_down_face: